src/Microsoft.Azure.WebJobs.Host/Common/OutArgumentBinding/CommonAsyncCollectorValueProvider.cs

```diff
@@ -6,20 +6,20 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host
 {
     // TUser is the parameter type from the user function we're binding to. 
     // TMessage is from the underying IAsyncCollector<TMessage>
     internal class CommonAsyncCollectorValueProvider<TUser, TMessage> : IOrderedValueBinder
```

Comment:
Agreed.  I paid less attention to the exact type and file layout when this was all non-public internal. 

---
In reply to: [54935398](https://github.com/Azure/azure-webjobs-sdk/pull/643#discussion_r54935398) [](ancestors = 54935398)